Makefile

```diff
@@ -59,8 +62,8 @@ dropdb:
 develop: APP_ENV=development
 develop: .state/docker-build-$(APP_ENV) serve
 
-develop-debug: APP_ENV=development
-develop-debug: .state/docker-build-$(APP_ENV) debug
+develop-shell: APP_ENV=development
+develop-shell: .state/docker-build-$(APP_ENV) shell
```

Comment:
I never saw this syntax for Makefiles. Does it mean that `APP_ENV=development` is only set (or overriden) for the `develop-shell` target?

Reply:
correct